ports/hidapi/portfile.cmake

```
@@ -1,11 +1,12 @@
-vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "linux" "osx" "uwp")
+vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "uwp")
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libusb/hidapi
     REF hidapi-0.10.1
     SHA512 0479706c631775483378070ff7170542725678eabc202a5bd07436c951fd766e01743417999ac3fb2b5436c865f6ace2cfced1f210fa3a3e88c19ceb3bbe0534
     HEAD_REF master
+    patch remove-duplicate-AC_CONFIG_MACRO_DIR.patch

```

```suggestion
    PATCHES remove-duplicate-AC_CONFIG_MACRO_DIR.patch
```
Maybe it is possible that the functions throws an error is you pass unknown options to it. And I wounder a lot why the pipeline was successful

---

Sorry for my wrong review.

---

Thanks for pointing this out. @autoantwort Sorry for my mistake.